pretyping/classops.ml

```diff
@@ -327,6 +327,9 @@ let message_ambig env sigma l =
   str"Ambiguous paths:" ++ spc () ++
   prlist_with_sep fnl (fun ijp -> print_path env sigma ijp) l
 
+let warn_ambiguous_paths env sigma =
+  CWarnings.create ~name:"ambiguous-paths" ~category:"typechecker" (message_ambig env sigma)
```

Comment:
CWarnings.create should not be in a lambda, instead do
~~~ocaml
let warn_ambiguous_paths = 
  CWarnings.create ~name:... ~category:... (fun (env,sigma,l) -> message_ambig env sigma l)
~~~
(I would also inline message_ambig)

Reply:
Could this have anything to do with the issue when locally disabling the warning?